api/resolvers/user_affiliations.py

```diff
@@ -0,0 +1,45 @@
+from graphql import GraphQLError
+
+from functions.error_messages import error_not_an_admin
+from functions.auth_wrappers import require_token
+from functions.auth_functions import (
+    is_super_admin,
+    is_admin,
+    is_user_write,
+    is_user_read
+)
+
+
+@require_token
+def resolve_test_user_claims(self, info, **kwargs):
+    """
+    This resolver returns the user_claims array -- A utility for testing
+    It requires that a JWT token be active, and that the user have an admin role
+    :returns: Returns the user_claims if user is an admin, raises error message if not.
+    """
+    roles = kwargs.get('user_roles')
+    test_role = kwargs.get('role')
+    org = kwargs.get('org')
+
+    if test_role == 'super_admin':
```

Comment:
Would this long if / else / elif statement make more sense to be a switch statement?

Reply:
It does make sense, sadly python dev team does not want us to have access to such beautiful things